chart/templates/_helpers.yaml

```diff
@@ -542,7 +542,11 @@ Create the name of the cleanup service account to use
 {{- end -}}
 
 {{ define "wait-for-migrations-command" }}
-  {{/* From Airflow 2.0.0 this can become [airflow, db, check-migrations] */}}
+  {{- if semverCompare ">=2.0.0" .Values.airflowVersion }}
```

Comment:
Should we add a simple test like the following:

```diff
diff --git a/chart/tests/test_webserver.py b/chart/tests/test_webserver.py
index 329e6117d..6f92af2d6 100644
--- a/chart/tests/test_webserver.py
+++ b/chart/tests/test_webserver.py
@@ -137,6 +137,21 @@ class WebserverDeploymentTest(unittest.TestCase):
             "image": "test-registry/test-repo:test-tag",
         } == jmespath.search("spec.template.spec.containers[-1]", docs[0])

+    @parameterized.expand(
+        [
+            ("2.0.0", ["airflow", "db", "check-migrations"]),
+            ("2.1.0", ["airflow", "db", "check-migrations"]),
+        ],
+    )
+    def test_wait_for_migration_airflow_version(self, airflow_version, expected_arg):
+        docs = render_chart(
+            values={
+                "airflowVersion": airflow_version,
+            },
+            show_only=["templates/webserver/webserver-deployment.yaml"],
+        )
+        assert expected_arg == jmespath.search("spec.template.spec.initContainers[0].args", docs[0])
+
     def test_should_add_extra_init_containers(self):
         docs = render_chart(
             values={
```

Reply:
Test added (and one for < 2.0 condition too.)